parse.go

```diff
@@ -114,7 +128,13 @@ func (p *parser) Parse(options ParseOptions) (*element.DataSet, error) {
 	// Change the transfer syntax for the rest of the file.
 	endian, implicit, err := p.parsedElements.TransferSyntax()
 	if err != nil {
-		return nil, err
+		if p.force {
+			// set the transfer sytanx values to defaults
+			endian = binary.LittleEndian
+			implicit = dicomio.ImplicitVR
```

Comment:
I'm not sure about this--the metadata elements are usually read in LittleEndian Explicit, which seems like a sane default to me. This might possibly be related to the read beyond buffer end issue, but I don't see a super strong connection.

Probably the right thing to do in the `force` case is just to read in some default transfer syntax until we parse the first element with a TransferSyntax tag, and then set that.

Reply:
You are correct. LittleEndian Explicit is the default for parsing meta data (see excerpt below). I have read that LittleEndian Implicit was the default for the rest of the file, but I didn't realize it would be different for the header.

"_Except for the 128 byte preamble and the 4 byte prefix, the File Meta Information shall be encoded using the Explicit VR Little Endian Transfer Syntax_ " [standard](http://dicom.nema.org/dicom/2013/output/chtml/part10/chapter_7.html)

My only concern for using LittleEndian Explicit as the default until we reach a transfer syntax tag, is that it seems all the data that is broken this way is also missing the entire header, and the TransferSyntax tag is contained in the header, which means we would never read a transfer syntax tag on actually broken files and would then parse the entire file with LittleEndian Explicit, which I think will cause problems (although I may be wrong about that).

For example:
"\# Dicom-File-Format

\# Dicom-Meta-Information-Header
\# Used TransferSyntax: Unknown Transfer Syntax

\# Dicom-Data-Set
\# Used TransferSyntax: Little Endian Implicit
(0008,0000) UL 128                                      #   4, 1 GenericGroupLength
(0008,0016) UI =SecondaryCaptureImageStorage            #  26, 1 SOPClassUID"